bukkit/src/main/java/dev/aurelium/auraskills/bukkit/skills/defense/DefenseAbilities.java

```diff
@@ -103,6 +105,10 @@ public void noDebuff(PotionSplashEvent event) {
     }
 
     private boolean isNegativeEffect(PotionEffectType type) {
+        if (VersionUtils.isAtLeastVersion(20, 6)) {
+            return (type.getCategory() == PotionEffectTypeCategory.HARMFUL);
```

Comment:
Small change: could you remove the unnecessary parentheses around this expression?

Reply:
Sorry, I meant the parenthesis right after the return statement itself. You should keep the curly braces for the if statement.